packages/fuselage/src/components/Card/CardControls.tsx

```diff
@@ -1,5 +1,12 @@
-const CardControls = ({ ...props }) => (
-  <div className='rcx-card__controls' {...props} />
+import type { ReactNode, MouseEvent } from 'react';
+
+type CardControlsProps = {
+  children?: ReactNode;
+  onClick?: (e?: MouseEvent<HTMLOrSVGElement>) => void;
```

Comment:
```suggestion
  onClick?: MouseEventHandler<HTMLDivElement>;
```

Reply:
Thanks for pointing this out too. A much better way. I have committed this change too.